funnel/views/index.py

```diff
@@ -116,13 +119,7 @@ def home(self) -> ReturnRenderWith:
                 else None
             ),
             'featured_profiles': [
-                p.current_access(datasets=('primary', 'related'))
-                for p in Profile.query.filter(
-                    Profile.is_verified.is_(True),
-                    Profile.organization_id.is_not(None),
-                )
-                .order_by(sa.func.random())
-                .limit(6)
+                p.access_for(roles={'all'}, datasets=('primary', 'related')) for p in featured_profiles if p
```

Comment:
The `if p` part can be removed now as the list is guaranteed to not have `None` in it